NEWS.md

```diff
@@ -1,5 +1,9 @@
 # serocalculator (development version)
 
+# serocalculator 1.2.9.9007
+
+* Add `as_noise_params` (#228) 
```

Comment:
Please move this line after line 8 (under `## New features`)

Reply:
This has been implemented. 
![image](https://github.com/user-attachments/assets/23860805-7650-408b-be55-fe4bc72c613a)

Reply:
This has been done.